LivlReviews.Api/Controllers/RequestController.cs

```diff
@@ -34,6 +49,35 @@ public ActionResult<Request> GetRequest(int id)
             return NotFound();
         }
         
+        return Ok(request);
+    }
+    
+    [HttpPost("{id}/approve")]
+    [UserIdClaim]
+    public async Task<ActionResult<Request>> ApproveRequest(int id)
+    {
+        var currentUserId = HttpContext.Items["UserId"] as string;
+        if(currentUserId is null) return Unauthorized();
+        
+        var currentUser = await userManager.FindByIdAsync(currentUserId);
+        if(currentUser is null)
+        {
+            return Unauthorized();
+        }
+
+        if (!Domain.Entities.Request.Can(currentUser.Role, Operation.UPDATE))
+        {
+            return Forbid();
+        }
+        
+        Request request = repository.GetById(id);
```

Comment:
We get a request, but I do not understand where is the code that say we approve it ? Have I missed something ?